Src/Core/OpenChat.Application/Users/UserNameAlreadyInUseException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace OpenChat.Application.Users
+{
+    public class UserNameAlreadyInUseException : Exception
+    {
+        public override string Message { get; }
+
+        public UserNameAlreadyInUseException(string message)
```

Comment:
`public UserNameAlreadyInUseException(string message) : base(message)`

Exception already contains a Message property and has a constructor specifically for it. And you can remove the Message property from this custom exception.